snowfakery/template_funcs.py

```diff
@@ -121,6 +121,7 @@ def try_parse_date(d):
             # swallow empty range errors per Python conventions
 
         def i18n_fake(self, locale: str, fake: str):
+            # deprecated by still here for backwards compatibility
```

Comment:
Ah, this answers my question above. Is there still a test that uses it so we are sure not to accidentally break it?

Reply:
coverage says yes.